Makefile

```diff
@@ -20,6 +20,10 @@ release: pre-build
 	@go build -ldflags "-s -w -X $(CODEPATH)/cmd.version=$(RESUMIC_VERSION) -X $(CODEPATH)/cmd.gitCommit=$(GITCOMMIT)" -o resumic
 	@packr2 clean
 
+nopackr: # Builds without installing packr. Useful if offline.
+	@go build -ldflags "-s -w -X $(CODEPATH)/cmd.version=$(RESUMIC_VERSION) -X $(CODEPATH)/cmd.gitCommit=$(GITCOMMIT)" -o resumic
```

Comment:
Probably a good idea to add a small line documenting what doesn't work in this build.

Reply:
I added this rule mostly for convenience, so that I could work while commuting without Internet access, so that's all it does.

Reply:
I'm not sure if adding a seprate command for every senario would be a good idea. We could use `-` before `go get`. This way even if the download failed it would still work.

Reply:
Adding to this. The makefile should only contain stuff that is applicable for most use cases and personal convenience is usually not one of them. Some projects even don't allow various additions to .gitignore such as editors, which especially for bigger projects might make them huge.

I see the reasoning behind this, but nopackr is not really discriptive. How about `raw-build`?

Further more:
* build depends on pre-build and raw-build
* raw-build is the build command

that should cut down on duplication, while still exposing the raw-build.

Hope that makes sense. If anyone doesn't like that approach, let me know.

Reply:
Gotcha, that makes sense! I can go ahead and remove it since it's not usually a situation that comes up often.